yal/assessments.py

```diff
@@ -65,7 +65,7 @@ async def state_survey_question(self):
 
         section = str(metadata.get("assessment_section", "1"))
 
-        assessment_name = metadata.get("assessment_name", "sexual_health_literacy")
+        assessment_name = metadata.get("assessment_name", "locus_of_control")
```

Comment:
This seems like a generic assessment application, why are we setting specific assessment names here?

Reply:
@KaitCrawford this is because its the first assessment, @rudigiesler can probably give more specifics?

Reply:
It was because this used to be for just one assessment, `sexual_health_literacy`. I didn't have time to go through and find everywhere where it was being called, and set the `assessment_name`, so I had it set to a default assessment if no specific assessment was set.